### ISSUE TITLE
Template variant (one-liner) from documentation examples not working

### ISSUE THREAD

**Issue by bcutter** [thomasloven/lovelace-auto-entities#304]
Regarding https://github.com/thomasloven/lovelace-auto-entities#examples:

✔ While this is working...
![grafik](https://user-images.githubusercontent.com/13799156/189500582-da15a067-0cfc-4cc2-935e-631ce5e8d9c5.png)

❌ ...this one...
![grafik](https://user-images.githubusercontent.com/13799156/189500590-ae2acbe9-0dcb-4ac3-a710-5a536af8d325.png)

...gives:

![grafik](https://user-images.githubusercontent.com/13799156/189500641-0fd529ff-5564-475d-9d96-ede5a54c924d.png)

That is pretty sad as I have a template (all my templates) written in that style.

**System information:**
- HA Core 2022.6.7
- Latest auto-entities

Please note that thes are working too:

✔
```
type: custom:auto-entities
card:
  type: entities
  title: Beleuchtung Test
filter:
  template: |
    {{
      expand(states.light)
      |selectattr('state', 'eq', 'on')
      |map(attribute='entity_id')
      |join(',')
    }}
```

✔
```
type: custom:auto-entities
card:
  type: entities
  title: Beleuchtung Test
filter:
  template: >-
    {{states.light | selectattr('state', '==', 'on') |
    map(attribute='entity_id') | join(',')}}
```

**To summarize:**
Using `| list` filter is working well in dev-tools template section, in template sensors and everywhere else - but not in auto-entities. A `| map(attribute='entity_id') | join(',')` filter instead works well.

Maybe that example is just outdated as it was working some time ago?

**Comment by thomasloven** [comment]
Fixed by #298?
